models/intel/device_support.md

```diff
@@ -17,18 +17,18 @@
 | driver-action-recognition-adas-0002-decoder | YES | YES | YES |
 | driver-action-recognition-adas-0002-encoder | YES | YES | YES |
 | emotions-recognition-retail-0003 | YES | YES | YES |
-| face-detection-0200 | YES |  | YES |
-| face-detection-0202 | YES |  | YES |
-| face-detection-0204 | YES |  | YES |
+| face-detection-0200 | YES | YES | YES |
+| face-detection-0202 | YES | YES | YES |
+| face-detection-0204 | YES | YES | YES |
 | face-detection-0205 | YES |  | YES |
```

Comment:
205/206 is not the same as 200..204?

Reply:
I don't know why 205/206 was GPU off in [PR](https://github.com/openvinotoolkit/open_model_zoo/pull/3261). We can add and try

Reply:
I missed them in my big PR and then gad to exclude one by one missed models